src/components/Invoices/InvoiceCard.jsx

```diff
@@ -1,5 +1,98 @@
-const InvoiceCard = () => {
-  return <div>My Invoices</div>;
+import { Stack, Grid, Paper, Box, Typography } from '@mui/material';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTheme } from '@mui/material/styles';
+import { useRouter } from 'next/router';
+
+const statusColor = {
+  ISSUED: '#64CEFB',
+  PAID: '#7DBE00',
+  OVERDUE: '#CF4520',
+};
+
+const month = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+const InvoiceCard = (props) => {
+  const newDate = props.due.split('/');
+  const getMonth = month[newDate[1] - 1];
```

Comment:
`getMonth` sounds like the name of a function. This is not a function.

```suggestion
  const month = months[newDate[1] - 1];
```